spec/models/collection_spec.rb

```diff
@@ -44,7 +46,17 @@
         allow(@collection).to receive(:members).and_return([file, file, file])
         allow(ActiveFedora::SolrService).to receive(:query).with(query, args).and_return(documents)
       end
-      it { is_expected.to eq 99 }
+
+      context "saved" do
```

Comment:
With `context` blocks, it can improve readability of output to start the string with "with" or "when", so:

```ruby
context "when saved" do
  # ...
end
```